perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -236,6 +236,8 @@ public AttributesManagerBlImpl(AttributesManagerImplApi attributesManagerImpl) {
 				memberResourceAttributeNames.add(attributeName);
 			} else if(attributeName.startsWith(AttributesManager.NS_USER_FACILITY_ATTR)) {
 				userFacilityAttirbuteNames.add(attributeName);
+			} else if(attributeName.startsWith(AttributesManager.NS_MEMBER_GROUP_ATTR)) {
```

Comment:
Please remove this, since this method doesn't provide member_group attributes and it belongs to the else branch where error is logged and silently skipped.